common/src/indexer_service/http/config.rs

```diff
@@ -5,11 +5,42 @@ use std::net::SocketAddr;
 
 use serde::{Deserialize, Serialize};
 use thegraph_core::{Address, DeploymentId};
+use tracing::warn;
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct DatabaseConfig {
     pub postgres_url: String,
 }
+impl DatabaseConfig {
+    pub fn format_db_config(
```

Comment:
Instead of a function here, `config` should have a `get_formatted_db_url(&self)-> Url`